Examples/Neutral_density_calculation.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>Please add in the method's doc that it also slices the data to a particular region?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>